consultation_analyser/support_console/views/consultations.py

```diff
@@ -1,20 +1,21 @@
+from django.contrib import messages
 from django.contrib.admin.views.decorators import staff_member_required
 from django.http import HttpRequest, HttpResponse
 from django.shortcuts import render
 
 from consultation_analyser.consultations import dummy_data, models
 from consultation_analyser.hosting_environment import HostingEnvironment
+from consultation_analyser.processing import run_processing_pipeline
 
 
 @staff_member_required
 def index(request: HttpRequest) -> HttpResponse:
-    # TODO - add messages
     if request.POST:
         try:
-            dummy_data.DummyConsultation(include_themes=False)
+            dummy_data.DummyConsultation(include_themes=False, number_questions=2)
+            messages.success(request, "A dummy consultation has been generated")
```

Comment:
This won't work as we'll need a redirect (and POST-then-redirect is the way to do this, I think?)

Reply:
This is working - could we have a chat about how messages work etc. because I am not sure I understand them properly.

Reply:
yes

Reply:
Experimenting with @nmenezes0, it seems that django will render messages on the same page if the view proceeds to render, or on the subsequent page if it redirects 🤷